steering-files/src/main/resources/org/hps/steering/recon/PhysicsRun2019_pass0_recon.lcsim

```diff
@@ -0,0 +1,439 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<lcsim xmlns:xs="http://www.w3.org/2001/XMLSchema-instance" xs:noNamespaceSchemaLocation="http://www.lcsim.org/schemas/lcsim/1.0/lcsim.xsd">
+    <!-- 
+      Steering file for running 2019 Reconstruction on readout data (LCIO or EVIO)
+      created:  9/24/19
+      @author Norman Graf <Norman.Graf@slac.stanford.edu>
+      @author PF <pbutti@slac.stanford.edu>
+    -->
+    <execute>
+        <driver name="EventMarkerDriver"/>
+        <!-- Enable the following if re-processing lcio files -->
+        <!--        <driver name="PreCleanupDriver"/>       -->
+        <!--  Event Reconstruction  -->
+        <!-- Skip events with known bad conditions -->
+        <!-- Not yet defined for 2019 data -->
+        <!--        <driver name="EventFlagFilter"/>  -->
```

Comment:
We do need the EventFlagFilter. The SVT conditions are added to the DB, so this should be made to work.

Reply:
Removed from steering file